backend/app/models/response_export.rb

```diff
@@ -12,20 +12,72 @@ def self.new_random_seed
     Random.new_seed
   end
 
+  scope :real, -> { where(is_testing: false) }
+  scope :by_date, -> { order(created_at: :desc) }
   scope :pending, -> { where(is_complete: false) }
+  scope :completed, -> { where(is_complete: true) }
 
   has_many_attached :files
 
   def ensure_fetched
     return if is_complete?
-    # TODO: kickoff and monitor Qualtrics download
-    raise 'not yet implemented' unless is_testing
 
-    generate_test_data
+    is_testing ? generate_test_data : fetch_real_responses
   end
 
   protected
 
+  def fetch_real_responses
+    # make sure we're not already fetching
+    api = QualtricsApi.new
+    latest = stage.response_exports.where.not(id: id).real.completed.by_date.last
+
+    progress_id = api.start_response_export(
+      stage.config['survey_id'], latest&.cutoff_at, cutoff_at
+    )
+
+    # TODO: monitor this and move it to a background job if it takes more than a few seconds
```

Comment:
This reminds me to ask you @nathanstitt , in my PR [here](https://github.com/openstax/kinetic/pull/238/files#diff-8a40967b7f6b2a2d4777f9b2d385020f5fd82b1eb6c1eb39297e73e054ef0095R112) the qualtrics cloning takes roughly a few seconds on the request. i wonder if this is also worth doing async? probably fine for now, but figured i'd mention it in case we have a more generic way to do that with your note here

Reply:
yes, we should do so then.  I think we can use background job in a simple thread